packages/ui-components/src/components/UITable/UITable.tsx

```diff
@@ -676,26 +676,31 @@ export class UITable extends React.Component<UITableProps, UITableState> {
      * @param newValue
      */
     private onTextInputChange(e: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>, newValue = ''): void {
-        const editedCell = this.state.editedCell || this.activeElement;
-        if (editedCell) {
-            editedCell.newValue = newValue;
-        }
-        this.setState({ editedCell });
-        const column = editedCell?.column;
-        if (column && typeof column.validate === 'function') {
-            this.validateCell(newValue);
-        }
-        if (this.props.renderInputs) {
-            this.saveCell(false, newValue);
-        }
+        this.setState((prevState) => {
+            const editedCell = prevState.editedCell ?? this.activeElement;
+            if (editedCell) {
+                editedCell.newValue = newValue;
+
+                const column = editedCell?.column;
+                if (column && typeof column.validate === 'function') {
+                    this.validateCell(newValue);
```

Comment:
I also see same react warning with new code:
![image](https://github.com/SAP/open-ux-tools/assets/90789422/a997ca8b-c5cb-42ee-a11a-2617061e5199)

If I apply old code - there no such warning

Scenario:
1. open storybook
2. open editable table
3. change value of any cell to `hello` - it should trigger validation, which applied as demo in story

Problem - react warning appears